ncm-metaconfig/src/main/pan/components/metaconfig/config.pan

```diff
@@ -9,6 +9,8 @@ include 'components/${project.artifactId}/schema';
 
 bind '/software/components/${project.artifactId}' = ${project.artifactId}_component;
 
+'version' ?= '${no-snapshot-version}';
```

Comment:
Using `?=` may actually be harmful for the version. Imagine you have two templates that both `include 'components/metaconfig/config'` but for some reason are name-spaced to different releases at different points in the compilation, `pkg_repl` would update the version installed, but `/software/components/metaconfig/version` would always reflect the first version included which would be confusing and not helpful if you are trying to use the version for logic.